corehq/apps/hqadmin/views/data.py

```diff
@@ -31,6 +32,12 @@ def to_json(doc):
     found_indices = {}
     es_doc_type = None
     for index in iter_index_cnames():
+        if not settings.IS_SAAS_ENVIRONMENT:
```

Comment:
@gherceg I have been running in circles in refactoring transient utils and was not able to make significant progress on it so I realised by the time I will do that it would be good to fix the page for users.